calico/networking/configuring/add-maglev-load-balancing.mdx

```diff
@@ -0,0 +1,71 @@
+---
+description: Add Maglev load balancing to a Kubernetes service.
+---
+
+# Add Maglev load balancing to a service
+
+This guide shows you how to enable Maglev load balancing for external traffic to a service.
+Maglev load balancing hashes the 5-tuple of packets destined for services in order to select a backend to load balance to.
+This opens up the possibility of using $[noderunning] as a distributed, fault-tolerant load balancer
+for externally-initiated connections, and allows for such connections to be preserved during a load balancer outage, provided another
+$[noderunning] is available to forward the connection instead.
```

Comment:
Generally, I like to keep this intro section easily intelligible to people not familiar with the area. I think we've lost a few readers by "hashes the 5-tuple of packets". 

We should be answering three questions in these short intros:
1) What's on this page?
2) What's this good for?
3) Is this for me?

The single great benefit that you mention is the fault tolerance. It's not going to drop packets.
Who benefits from that? Everybody, I guess. But probably it's most worth the bother for certain types of clusters or workloads. What are those?